### PR TITLE
Component prototype, Refactored Into Styled Components, Styled To Match Zillow.com

### PR DESCRIPTION
**Pull request by spraguejdev**
I've basically styled the component to look like zillow.com. It is not 100% complete just yet, I would like to hear your thoughts before I refactor and move forward. 

Thanks!

### CODE REVIEW

File: client/src/components/App.jsx [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+        this.setState({
+          propertyData: data.propertyData, 
+          comparableHomesData: data.comparableHomesData,
+          localHomesData: data.localHomesData
+        });
+      });
+  }
+  
+  render () {
+    // console.log(this.state.propertyData[0].id)
```
Comment (leightonchen) [inline client/src/components/App.jsx]: Don't forget to remove extra testing logs like this.

File: client/src/components/ZestimateAdditionalInfo.jsx [hunk 2]
```diff
@@ -1,55 +1,80 @@
...
+
+library.add(faHome, faCircle, faAngleDown);
+
+const AdditionalZestimateInformation = styled.div`
+display: flex;
+`;
+const ZestDeepDive = styled.div``;
+
+const ZestDeepDiveIntro = styled.div`
+  text-align: left;
```
Comment (leightonchen) [inline client/src/components/ZestimateAdditionalInfo.jsx:18]: Inconsistent tabbing with styled-components. Should either have all of them tabbed once, or not tabbed at all. Other .jsx files also have some similar issues.

**Review by leightonchen — CHANGES_REQUESTED** [review]
Update your `.gitignore`! It shouldn't include `bundle.js`, or any images like `id.jpg`. 

Also: getting an error when I run `npm run react-dev`. Because of this, I was not able to confirm it all works.

```
lei@LAPTOP-NMDEKGDS MINGW64 ~/Documents/hrsf/john-sprague-component-home-value (component-prototype)
$ npm run react-dev

> john-sprague-component-home-value@1.0.0 react-dev C:\Users\lei\Documents\hrsf\john-sprague-component-home-value
> webpack -d --watch --display-error-details

C:\Users\lei\Documents\hrsf\john-sprague-component-home-value\node_modules\webpack-cli\bin\cli.js:231
                                throw err;
                                ^

Error: custom keyword definition is invalid: data/errors should be boolean
    at Ajv.addKeyword (C:\Users\lei\Documents\hrsf\john-sprague-component-home-value\node_modules\ajv\lib\keyword.js:65:13)
    at module.exports (C:\Users\lei\Documents\hrsf\john-sprague-component-home-value\node_modules\ajv-errors\index.js:10:7)
    at Object.<anonymous> (C:\Users\lei\Documents\hrsf\john-sprague-component-home-value\node_modules\schema-utils\src\validateOptions.js:22:1)
    at Module._compile (C:\Users\lei\Documents\hrsf\john-sprague-component-home-value\node_modules\v8-compile-cache\v8-compile-cache.js:178:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (C:\Users\lei\Documents\hrsf\john-sprague-component-home-value\node_modules\v8-compile-cache\v8-compile-cache.js:159:20)
    at Object.<anonymous> (C:\Users\lei\Documents\hrsf\john-sprague-component-home-value\node_modules\schema-utils\src\index.js:7:25)
    at Module._compile (C:\Users\lei\Documents\hrsf\john-sprague-component-home-value\node_modules\v8-compile-cache\v8-compile-cache.js:178:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (C:\Users\lei\Documents\hrsf\john-sprague-component-home-value\node_modules\v8-compile-cache\v8-compile-cache.js:159:20)
    at Object.<anonymous> (C:\Users\lei\Documents\hrsf\john-sprague-component-home-value\node_modules\webpack\lib\SourceMapDevToolPlugin.js:13:25)
    at Module._compile (C:\Users\lei\Documents\hrsf\john-sprague-component-home-value\node_modules\v8-compile-cache\v8-compile-cache.js:178:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (C:\Users\lei\Documents\hrsf\john-sprague-component-home-value\node_modules\v8-compile-cache\v8-compile-cache.js:159:20)
    at Object.<anonymous> (C:\Users\lei\Documents\hrsf\john-sprague-component-home-value\node_modules\webpack\lib\WebpackOptionsApply.js:16:32)
    at Module._compile (C:\Users\lei\Documents\hrsf\john-sprague-component-home-value\node_modules\v8-compile-cache\v8-compile-cache.js:178:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! john-sprague-component-home-value@1.0.0 react-dev: `webpack -d --watch --display-error-details`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the john-sprague-component-home-value@1.0.0 react-dev script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\lei\AppData\Roaming\npm-cache\_logs\2019-02-14T19_35_26_078Z-debug.log
```

File: client/src/components/App.jsx (hunk 1 above)
Comment (spraguejdev) [reply client/src/components/App.jsx]: Thanks. Done!

File: client/src/components/ZestimateAdditionalInfo.jsx (hunk 2 above)
Comment (spraguejdev) [reply client/src/components/ZestimateAdditionalInfo.jsx:18]: @leightonchen Good eye

**Review by leightonchen — APPROVED** [review]

**Pull request closed without merging**